flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -469,6 +479,58 @@ void TizenWindowEcoreWl2::Show() {
   ecore_wl2_window_show(ecore_wl2_window_);
 }
 
+void TizenWindowEcoreWl2::UpdateFlutterCursor(const std::string& kind) {
+#ifdef TV_PROFILE
+  int pointer_size = -1;
+  if (vconf_get_int(kSysMouseCursorPointerSizeVConfKey, &pointer_size) < 0) {
+    FT_LOG(Info) << "Failed to load cursor size.";
+  }
+
+  std::string cursor_name = "normal_default";
+  if (kind == "basic") {
+    if (pointer_size == 0) {  // Large.
+      cursor_name = "large_normal";
+    } else if (pointer_size == 1) {  // Medium.
+      cursor_name = "medium_normal";
+    } else if (pointer_size == 2) {  // Large.
+      cursor_name = "small_normal";
+    } else {
+      cursor_name = "normal_default";
+    }
+  } else if (kind == "click") {
+    if (pointer_size == 0) {  // Large.
+      cursor_name = "large_normal_pnh";
+    } else if (pointer_size == 1) {  // Medium.
+      cursor_name = "medium_normal_pnh";
+    } else if (pointer_size == 2) {  // Large.
+      cursor_name = "small_normal_pnh";
+    } else {
+      cursor_name = "normal_pnh";
+    }
+  } else if (kind == "text") {
+    if (pointer_size == 0) {  // Large.
+      cursor_name = "large_normal_input_field";
+    } else if (pointer_size == 1) {  // Medium.
+      cursor_name = "medium_normal_input_field";
+    } else if (pointer_size == 2) {  // Large.
+      cursor_name = "small_normal_input_field";
+    } else {
+      cursor_name = "normal_input_field";
+    }
+  } else if (kind == "none") {
+    cursor_name = "normal_transparent";
```

Comment:
Is there any other way to unshow the cursor? It would be helpful if we could support hiding the cursor (`SystemMouseCursors.none`) on non-TV profiles so that users can use their own software cursors.